### PR TITLE
Add "More..." chapter

### PR DESCRIPTION
**Pull request by fioddor**
* Add a chapter providing useful pointers for those who want to dive deeper into the metrics subject.
* Link to the CHAOSS community, as it is very relevant for open source metrics around open source development: It is a welcoming community with wide and varied participation. They discuss and develop metrics for open source development and develop open source tools to manage them. It's just the right place to go for help or knowledge.

# DISCLAIMER:
I fear I might be violating the CONTRIBUTING.md where it says _"Please don't add references or links to things you or your employer have created"_ because I currently work for a company that joined that community and contributed some of their tools. But I really think this is **very relevant and valuable information for the readers** and I'm pointing to the wider community, not my specific company or its tools.

I thought of creating an issue for others to consider and include the information I suggest, and thereby circumvent this conflict of interest. But it seems the issues are disabled and the link leads to the _pull request_ page, so I assume a PR is the valid way/place to have this conversation.

-----
- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Comment by Rk78678** [comment]
Hi

**Comment by Rk78678** [comment]
P

**Comment by Witte1979** [comment]
Hou stop ik en krijg ik geen mails meer van girhub

Op wo 9 feb. 2022 om 01:26 schreef Rk78678 ***@***.***>

File: _articles/metrics.md [hunk 1]
```diff
@@ -124,3 +124,7 @@ You could also measure the time it takes to move between stages in the contribut
 ## Use 📊 to learn about people
 
 Understanding metrics will help you build an active, growing open source project. Even if you don't track every metric on a dashboard, use the framework above to focus your attention on the type of behavior that will help your project thrive.
+
+## More...
+
+There's a welcoming open source community around open source metrics, called [CHAOSS](https://chaoss.community/), in case you have doubts or want to dive deeper. They develop metrics and provide free software tools.
```
Comment (MikeMcQuaid) [inline _articles/metrics.md:130]: How about something like this:

```suggestion

[CHAOSS](https://chaoss.community/) is a welcoming, open source community focused on analytics, metrics and software for community health.
```

**Review by BinHavin0323 — COMMENTED** [review]
>

**Review by MikeMcQuaid — APPROVED** [review]
Thanks so much for your first contribution! Without people like you submitting PRs we couldn't run this project. You rock, @fioddor!

File: _articles/metrics.md (hunk 1 above)
Comment (Aftabalamkhan120) [reply _articles/metrics.md]: @misc{ wiki:xxx,
   author = "Meta",
   title = "Celebrate Women --- Meta{,} discussion about Wikimedia projects",
   year = "2022",
   url = "\url{https://meta.wikimedia.org/w/index.php?title=Celebrate_Women&oldid=22984427}",
   note = "[Online; accessed 19-March-2022]"
 }

**Pull request merged**